server/src/main/java/org/candlepin/guice/CandlepinModule.java

```diff
@@ -148,6 +137,24 @@
 import com.google.inject.name.Names;
 import com.google.inject.persist.jpa.JpaPersistModule;
 
+import org.hibernate.cfg.beanvalidation.BeanValidationEventListener;
+import org.hibernate.validator.HibernateValidator;
+import org.hibernate.validator.HibernateValidatorConfiguration;
+import org.quartz.JobListener;
+import org.quartz.spi.JobFactory;
+import org.xnap.commons.i18n.I18n;
+
+import java.util.Properties;
```

Comment:
> What is the correct order? From the candlepin.importorder it seems that Eclipse is ordering the imports correctly.

:frowning: Sorry!  Your commit corrected someone else's ordering error and I wasn't paying attention!  We should probably add `io` as an entry in the importorder file though so that the `javax` and `java` packages remain at the bottom.